Tools/Scripts/libraries/reporelaypy/reporelaypy/webserver.py

```diff
@@ -76,6 +76,12 @@ def health():
         mimetype='application/json',
     )
 
+
+@app.route('/compare')
+def compare():
+    return render_template('compare.html')
+
+
```

Comment:
I don't see a landing page here. Won't we continue using the redirect?

Reply:
Updated so this will be the landing page for commits.webkit.org!